modules/core/src/config.ts

```diff
@@ -193,7 +193,7 @@ export const krsProviders = {
   dai: {
     feeType: 'flatUsd',
     feeAmount: 0, // dai will receive payments off-chain
-    supportedCoins: ['btc', 'eth', 'xlm', 'xrp', 'dash', 'zec', 'ltc', 'bch']
+    supportedCoins: ['btc', 'eth', 'xlm', 'xrp', 'dash', 'zec', 'ltc', 'bch', 'bsv', 'bcha']
```

Comment:
So the two bitcoin cash forks are going to be `bch` and `bcha`? For somer reason I thought `bcha` meant Bitcoin ABC, and the other fork will be called BCHN ? And then, is there a way to verify that "dai" (which I figured out is now called "Coincover") operates on these two "new" coins?

Reply:
BCHN will inherent the existing BCH's nomenclature. We are talking to coincover to help us test this!